doc/multi-o.md

```diff
@@ -0,0 +1,117 @@
+# Multi Orchestrator Setup
+
+This document describes the various ways an orchestrator could run multiple nodes and separate concerns. 
+
+## Unlocking Account and Connecting to Ethereum
+
+Unlocking your Ethereum account and connecting to a JSON-RPC provider are necessary steps to run the Livepeer node unless it's in standalone transcoder mode. 
+
+For brevity we'll exclude these flags in the examples by `<...ETH SETUP...>` but these should always be required. If no Ethereum account is available the Livepeer node will create one for you.
+
+```
+livepeer
+    -network <mainnet|rinkeby> \
+    -ethUrl <JSON_RPC_PROVIDER> \
+    -ethKeystorePath <PATH_TO_KEY> \
+    -ethAcctAddr <ETHEREUM_ADDRESS> \
+    -ethPassword <ETHEREUM_ACCOUNT_PASSWORD (string|path)> \
+    -ethController <CONTROLLER_CONTRACT_ADDRESS (required if '-network' not provided)>
+```
+
+## Single Orchestrator with Redeemer
+
+This allows an orchestrator running a single on-chain node to use a seperate Ethereum account to redeem winning tickets on-chain and pay for that transaction while the recipient is still orchestrator node's Ethereum address.
+
+The orchestrator node will still be responsible for initialising rounds and calling `reward`. 
+
+1. Start the Redeemer 
+
+```shell
+livepeer
+    <...ETH SETUP...\
+    -redeemer=true \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port \ 
```

Comment:
Missing > at the end. Same comment applies for a few of the other flag values in this doc so it'd be good to take a pass through all them to add the missing > at the end.

Reply:
fixed in 4131a02b